components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -648,17 +649,22 @@ void CosmeticFiltersJSHandler::OnHiddenClassIdSelectors(
   TRACE_EVENT1("brave.adblock", "OnHiddenClassIdSelectors", "url", url_.spec());
 
   base::Value::List* hide_selectors = result.FindList("hide_selectors");
-  DCHECK(hide_selectors);
+  if (!hide_selectors) {
+    return;
+  }
```

Comment:
let's also not early return here, and `if`-gate the related functionality below instead

Reply:
I've fixed the previous "if" b/c it is simple, but other than that i'd prefer to keep this PR super trivial and quick just to fix the crash as i'm not the author of the code and also apparently there is lack of tests (as the crash hasn't been caught). Would like to avoid writing new "if"s here and there

Reply:
sure, ok makes sense